home-manager.nix

```diff
@@ -317,6 +352,43 @@ in
           in
           ''
             if [[ -n ''${mountedPaths[${mountPoint}]+x} ]]; then
+                triesLeft=3
+                # Check to avoid trying to unmount a symlink
+                if ${mount} | grep -F ${mountPoint}' ' >/dev/null; then
```

Comment:
Git messed up the diff, but the addition of this `if` is the actual change to `mkUnmount`.